02-Architecture Constraints.md

```diff
@@ -10,4 +10,7 @@ Technical constraints that apply to the design and development of the system.
 
 Any Wikibase entity data displayed as part of article content has to be loadable by a [Lua](./12-Glossary.md/#Lua) function.
 
+## [MediaWiki Extensions](./12-Glossary.md/#mediawiki-extension )
 
+Any functionality that doesn't belong in [mediawiki core](https://www.mediawiki.org/wiki/Core) has to be registered as a mediawiki extension.
+**Mediawiki extensions cannot have extensions of their own.**
```

Comment:
We briefly discussed this and I'm not sure there's a lot of value for us in this sentence. While technically everything is a Mediawiki extension, we can of course create an extension that provides additional functionality only to the Wikibase extension, so that it will have no effect if Wikibase is not there.

In fact, I think it would even help us to make the distinction between "Mediawiki extensions" and "Wikibase extensions", as Adam did in the list of components, even if technically there is no such thing.

Reply:
+1

Reply:
I mean, they totally can? they just don't?